app/views/places/feed.atom.builder

```diff
@@ -0,0 +1,37 @@
+def place_to_html_description(place)
```

Comment:
We shouldn't define a method in a view. We could use a partial for this, or maybe a presenter.